src/application.py

```diff
@@ -46,4 +49,36 @@ async def on_ready():
     logger.info("Bot is ready")
 
 
+@client.command(name='reload', help='To reload all cogs.')
+async def reload(ctx):

```

Comment:
Suggestion: Accept an optional cog_name. If provided, reload only that cog, otherwise reload all cogs